frontend/src/time_utils.js

```diff
@@ -8,8 +8,8 @@ export function convertTimestampToDate(timestampDate) {
     .toDateString().slice(4);
 }
 
-/** Format JavaScript date in the form MM-DD-YYYY. */
-export function formatDate(date) {
+/** Format JavaScript date in the form MM/DD/YYYY if mFirst is true, otherwise YYYY/MM/DD. */
+export function formatDate(date, mFirst) {
   var d = new Date(date),
       month = '' + (d.getMonth() + 1),
```

Comment:
why do you append the `' '` before day and month, but not before year?

Reply:
I think it's to be able to add the '0' in front of months/days that are single digits